Diana_Mokhnitskaya/HW6/HW6_task3.py

```diff
@@ -0,0 +1,9 @@
+import re
+from collections import Counter
+with open("text.txt") as file:
+    new_string = file.read()
+    print(new_string)
+    all_characters = re.findall(r"[a-zA-Z]\D\S", new_string)
+    print(Counter(new_string))
+
+# Не понимаю, как убрать символы и точки из подсчета
```

Comment:
all_characters = re.findall(r"[A-z]", a)
print(Counter(all_characters))

Reply:
Update in new commit